### PR TITLE
Develop Reply and Fix Post test case

### PR DESCRIPTION
**Pull request by JeonJinil**

### CODE REVIEW

File: src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @ManyToOne
+    @JoinColumn(name="Post_Id")
+    private Post post;
+
+    @ManyToOne
+    @JoinColumn(name="User_Id")
+    private User user;
+
+    @Column
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java:33]: How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

File: src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import javax.persistence.*;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class Reply extends BaseTimeEntity {
+
+    @javax.persistence.Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java:20]: How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

File: src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Column(length = 500, nullable = false)
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java:22]: How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

File: src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    private Post post;
+
+    @ManyToOne
+    @JoinColumn(name="User_Id")
+    private User user;
+
+    @Column
+    private Long replyParentId;
+
+    @Column
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java:36]: How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

File: src/main/java/com/sweetwith/dailynote/domain/like/PostLike.java [hunk 5]
```diff
@@ -22,7 +22,7 @@
     private Post post;
 
     @ManyToOne
-    //@JoinColumn(name="User_Id")
+    @JoinColumn(name="User_Id")
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/like/PostLike.java:25]: I think id's column is Id as declared variable in User class due to not specifying.

https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

File: src/main/java/com/sweetwith/dailynote/domain/posts/Post.java [hunk 6]
```diff
@@ -26,10 +26,9 @@
     private String content;
 
     @ManyToOne
-    //@JoinColumn(name="User_Id")
+    @JoinColumn(name="User_Id")
```
Comment (kcassia) [inline src/main/java/com/sweetwith/dailynote/domain/posts/Post.java:29]: I think id's column is Id as declared variable in User class due to not specifying.

https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507

**Review by kcassia — COMMENTED** [review]
Review to column name.

File: src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java (hunk 1 above)
Comment (JeonJinil) [reply src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java:33]: you means, user value should be clearly declare using @Column annotation?

**Pull request merged**